src/DependencyInjection/Configuration.php

```diff
@@ -45,30 +45,51 @@ public function getConfigTreeBuilder(): TreeBuilder
-                                            ->booleanNode('extend')->defaultFalse()->end()
-                                            ->arrayNode('gravity')
+                                            ->enumNode('resizing_type')
+                                                ->values(['fit', 'fill', 'fill-down', 'force', 'auto'])
+                                                ->defaultValue('fit')
+                                            ->end()
+                                            ->integerNode('width')->defaultValue(0)->min(0)->end()
+                                            ->integerNode('height')->defaultValue(0)->min(0)->end()
+                                            ->booleanNode('enlarge')->end()
+                                            ->arrayNode('extend')
                                                 ->children()
-                                                    ->enumNode('type')
-                                                        ->values(['no', 'so', 'ea', 'we', 'noea', 'nowe', 'soea', 'sowe', 'ce'])
-                                                        ->defaultValue('ce')
+                                                    ->booleanNode('extend')->defaultFalse()->end()
+                                                    ->arrayNode('gravity')
+                                                        ->children()
+                                                            ->enumNode('type')
+                                                                ->values(['no', 'so', 'ea', 'we', 'noea', 'nowe', 'soea', 'sowe', 'ce'])
+                                                                ->defaultValue('ce')
+                                                            ->end()
+                                                            ->integerNode('x_offset')->defaultValue(0)->end()
+                                                            ->integerNode('y_offset')->defaultValue(0)->end()
+                                                        ->end()
                                                     ->end()
-                                                    ->integerNode('x_offset')->defaultValue(0)->end()
-                                                    ->integerNode('y_offset')->defaultValue(0)->end()
                                                 ->end()
                                             ->end()
                                         ->end()
                                     ->end()
+                                    ->arrayNode('rotate')
+                                        ->children()
+                                            ->enumNode('angle')
+                                                ->values([0, 90, 180, 270])
+                                            ->end()
+                                        ->end()
+                                    ->end()
+                                    ->arrayNode('height')
```

Comment:
Should be improved as all option that can be used in standalone can be accepted (resize_type for example can be used alone in doc)